example/fine_advection/advection_driver.cpp

```diff
@@ -89,18 +102,22 @@ TaskCollection AdvectionDriver::MakeTaskCollection(BlockList_t &blocks, const in
     auto start_send = tl.AddTask(none, parthenon::StartReceiveBoundaryBuffers, mc1);
     auto start_flxcor = tl.AddTask(none, parthenon::StartReceiveFluxCorrections, mc0);
 
-    static auto desc =
-        parthenon::MakePackDescriptor<advection_package::Conserved::scalar>(
-            pmesh->resolved_packages.get(), {parthenon::Metadata::WithFluxes},
-            {parthenon::PDOpt::WithFluxes});
+    using namespace advection_package::Conserved;
+    static auto desc = parthenon::MakePackDescriptor<phi>(
+        pmesh->resolved_packages.get(), {parthenon::Metadata::WithFluxes},
+        {parthenon::PDOpt::WithFluxes});
     using pack_desc_t = decltype(desc);
 
-    static auto desc_fine =
-        parthenon::MakePackDescriptor<advection_package::Conserved::scalar_fine>(
-            pmesh->resolved_packages.get(), {parthenon::Metadata::WithFluxes},
-            {parthenon::PDOpt::WithFluxes});
+    static auto desc_fine = parthenon::MakePackDescriptor<phi_fine>(
```

Comment:
`phi_fine` could be packed in the same pack as `phi`, right?  This is just a convenience here so that you can assign `pack_desc_fine_t` which is later used as a template argument (e.g., `Stokes`)?

Reply:
yes, there is nothing preventing you from packing fine and regular variables in the same pack. It is just not desirable here.